Jenkinsfile

```diff
@@ -0,0 +1,56 @@
+pipeline {
+        agent any
+        environment {
+      	           GIT_HASH = GIT_COMMIT.take(7)
+		   Version_ID = "$BUILD_NUMBER" + "-" + "$GIT_HASH" + "-" + "$BUILD_TIMESTAMP"
+        }
+        stages{
+                   stage('Install dependencies') {
+                         steps {
+                              sh 'npm i'
+		              echo 'Building..'
+                              echo "Running job: ${env.JOB_NAME}\n Build: ${env.BUILD_ID} - ${env.BUILD_URL}\nPepeline: ${env.RUN_DISPLAY_URL}"
+                         }
+                   }   
+                    stage('Build') {
+                          steps {
+                               sh 'npm run build'
+                         }
+                   }
+                    stage('Export') {
+                          steps {
+                               sh 'npm run export'
+                          }
+                   }
+                    stage('Version Number') {
+                          steps {
+			       echo "1.0.$Version_ID"
+                          }
+                   }
+	            stage('Upload Master Branch Artifact Repo') { 
+	                  when {
+		               branch 'master'
```

Comment:
Currently, our main branch is called "main", not "master"
<img width="531" alt="image" src="https://user-images.githubusercontent.com/61776449/168410468-3a9866e9-21e8-4730-bb21-b9b232fba274.png">